.golangci.yml

```diff
@@ -32,6 +32,8 @@ linters:
     - goerr113
     - exhaustivestruct
     - lll
+    - wsl
+    - gofumpt
```

Comment:
What's the issue with these linters? They're pretty strict, reason is to enforce a coding style that is constant across all the files.